tests/tests_wireless_wpa3_owe_nm.yml

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# This file was generated by ensure_provider_tests.py
+---
+# set network provider and gather facts
+- hosts: all
+  name: Run playbook 'playbooks/tests_wireless_wpa3_owe.yml' with nm as provider
+  tasks:
+    - name: Set network provider to 'nm'
+      set_fact:
+        network_provider: nm
+      tags:
+        - always
+
+
+# The test requires or should run with NetworkManager, therefore it cannot run
+# on RHEL/CentOS 6
```

Comment:
This comment does not explain `ansible_distribution_major_version != '7'`

Reply:
@tyll , I realized that the following comment was never written into `tests_*_nm.yml`, do we need to create a separate PR to make sure that comment was also properly generated when running `python ensure_provider_tests.py generate`

Reply:
yes, good idea.

Reply:
PR for enabling comment support, https://github.com/linux-system-roles/network/pull/390

Reply:
@cathay4t , included comment for explaining ansible_distribution_major_version != '7'

Reply:
@cathay4t , included comment for explaining ansible_distribution_major_version != '7'